src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -597,63 +856,109 @@ export default function createIntegration() {
 }
 ```
 
-The `astro/env/setup` module allows you to provide an implementation for `getSecret()`. In your server entrypoint, call `setGetEnv()` as soon as possible:
+### `experimentalStaticHeaders`
 
-```js ins={2,4}
-import { App } from 'astro/app';
-import { setGetEnv } from "astro/env/setup"
+<p>
 
-setGetEnv((key) => process.env[key])
+**Type:** `boolean`<br />
+<Since v="5.9.3" />
+</p>
 
-export function createExports(manifest) {
-  const app = new App(manifest);
+When this feature is enabled, Astro will return a map of the `Headers` emitted by the static pages. This map `experimentalRouteToHeaders` is available in the [`astro:build:generated` hook](/en/reference/integrations-reference/#astrobuildgenerated).
+  
+The value of the headers might change based on the features enabled/used by the application.
 
```

Comment:
```suggestion
```
I think I'd keep these as one paragraph. (I can't make a suggestion across a deleted line, but I can remove the line between them.)

I also might put that paragraph *below* the code example, so it's not as far removed from the definition.